Source/OCMock/OCMock.h

```diff
@@ -79,6 +79,7 @@
 ({ \
     _Pragma("clang diagnostic push") \
     _Pragma("clang diagnostic ignored \"-Wunused-value\"") \
+    _Pragma("clang diagnostic ignored \"-Wunused-getter-return-value\"") \
```

Comment:
On Xcode 6.2 I'm seeing  "Unknown warning group '-Wunused-getter-return-value', ignored." with this addition. Do you silence this warning in your setup?